docs/v1/P4Runtime-Spec.mdk

```diff
@@ -1327,6 +1352,17 @@ Above we see several different types of annotations:
 * `@description` - This will populate the `PkgInfo.doc.description` message
   field
 
+* `@platform_property` - This is used to populate a specific field within the
+  `PlatformProperty` message in `PkgInfo.platform_property`. Multiple
+  `@platform_property` annotations are allowed. For compactness,
+  multiple key-value pairs can appear in a single `@platform_property`
+  annotation, separated by commas. Each key must only appear once and the
+  compiler must reject the program if one appears multiple times. The `key`s
```

Comment:
Nit: Is "key" referring to code (a variable / proto field) here? If not, let's not put it in backticks.

Reply:
It is, yeah. It's referring to the code example above.